app.py

```diff
@@ -414,100 +365,144 @@ async def github_metrics():
         }
         github_data.append(data)
 
-    supabase_client = SupabaseInterface()
-    data = supabase_client.add_github_metrics(github_data)
-    return data.data
+    data =  await PostgresORM().add_github_metrics(github_data)
+    return data
 
 @app.route('/job_classroom')
 async def my_scheduled_job_test():
```

Comment:
Not sure if this is required. Please confirm with Mohit. We are not handling classrooms anymore.